<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatabaseViewer</class>
 <widget class="QTabWidget" name="databaseViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TabWidget</string>
  </property>
  <property name="currentIndex">
   <number>0</number>
  </property>
  <widget class="QWidget" name="dbTab">
   <attribute name="title">
    <string>Symbol Database</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="symbolFilter"/>
      </item>
      <item>
       <widget class="QLineEdit" name="dirFilter"/>
      </item>
      <item>
       <widget class="QLineEdit" name="fileFilter"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="dbView"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="historyTab">
   <attribute name="title">
    <string>History</string>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTableView" name="historyView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
